src/components/InputFilters/InputFilters.jsx

```diff
@@ -7,11 +7,14 @@ function InputFilters({ isOpen, setFilteredText }) {
   const inputRef = useRef();
 
   useEffect(() => {
-    if (isOpen) inputRef.current?.focus();
+    if (isOpen) {
+      inputRef.current?.focus();
+    }
   }, [ isOpen ]);
 
   return (
     <Form.Control
+      id="filters-input"
```

Comment:
One should not user id's for styling. Because ID should be unique in the DOM, it implies that no other element may ave that style. It should instead be something like `.filter-form-background`